setup.py

```diff
@@ -53,4 +52,13 @@
             'token_revoke = wazo_auth_cli.commands.token:TokenRevoke',
         ],
     },
+    install_requires=[
+        "wazo-lib-rest-client@git+https://github.com/wazo-platform/wazo-lib-rest-client.git"
+        "@master",
```

Comment:
feel free to use `# noqa <code>` or encapsulate the split in paranthese
```suggestion
        (
            "wazo-lib-rest-client@git+https://github.com/wazo-platform/wazo-lib-rest-client.git"
            "@master"
        ),
```